dali-rp2.pyproj

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{21de6965-e7a9-45bf-bc35-9aafe37f7ddf}</ProjectGuid>
+    <ProjectHome />
+    <StartupFile>src\dali\plugin\country\us.py</StartupFile>
+    <SearchPath />
+    <WorkingDirectory>.</WorkingDirectory>
+    <OutputPath>.</OutputPath>
+    <ProjectTypeGuids>{888888a0-9f3d-457c-b088-3a5042f75d52}</ProjectTypeGuids>
+    <LaunchProvider>Standard Python launcher</LaunchProvider>
+    <InterpreterId>MSBuild|.venv|$(MSBuildProjectFullPath)</InterpreterId>
+    <CommandLineArguments>-t 1 -s -o C:\Users\cyates\Documents\rp2-files\cyates\dali-output -p cyates- C:\Users\cyates\Documents\rp2-files\cyates\cyates.ini</CommandLineArguments>
+    <EnableNativeCodeDebugging>False</EnableNativeCodeDebugging>
+    <TestFramework>pytest</TestFramework>
+    <UnitTestPattern>test*.py</UnitTestPattern>
+    <UnitTestRootDirectory>tests</UnitTestRootDirectory>
+    <Environment>LOG_LEVEL=DEBUG</Environment>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'" />
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'" />
+  <PropertyGroup>
+    <VisualStudioVersion Condition=" '$(VisualStudioVersion)' == '' ">10.0</VisualStudioVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="config\test_config.ini" />
+    <Content Include="input\test_ods_rp2_input.ini" />
+    <Content Include="mypy.ini" />
+    <Content Include="src\dali_rp2.egg-info\dependency_links.txt" />
+    <Content Include="src\dali_rp2.egg-info\entry_points.txt" />
+    <Content Include="src\dali_rp2.egg-info\requires.txt" />
+    <Content Include="src\dali_rp2.egg-info\SOURCES.txt" />
+    <Content Include="src\dali_rp2.egg-info\top_level.txt" />
```

Comment:
These files are generated: I'm not sure they should be included in an IDE config file.

Reply:
When I created the project I used Microsoft's "import" existing python project.  That's probably why it included these. I've taken them out.